### PR TITLE
HLE: Register cellSysutilNpEula, libmedi, cellCrossController

### PR DESCRIPTION
**Pull request by AniLeo**
**cellSysmodule: Register 0xF044 module (cellSysutilNpEula)**
Found by debugging Uncharted 2 Demo (NPEA90055)
Helps in all games that register sys module configuration 'multi-player'

**cellSysmodule: Register 0x0054 module (libmedi)**
Found on Motorstorm Apocalypse [NPEA00315] (thanks Zangetsu for the log)

**cellSysmodule: Register 0x005C module (cellCrossController)**
Found on LittleBigPlanet 2 [BCES00850] (thanks Zangetsu for the log)

### CODE REVIEW

**Comment by Zangetsu38** [comment]
oh thx, i have found 0xF044 but i not find correct name module, and if you have any idea for 0x005C and 0x0054

**Comment by AniLeo** [comment]
@Zangetsu38 What are the games they're used on? Only way to find those names is to debug the games, I can give it a shot.

**Comment by Zangetsu38** [comment]
hmm i don't remember :/

**Comment by darkredflame** [comment]
I'm pretty sure that Motorstorm Apocalypse tries to load module 0x0054 - but I don't have the game currently ripped to test it out.

**Comment by AniLeo** [comment]
@Zangetsu38 https://github.com/Zangetsu38/Log-Rpcs3/blob/master/BD/EUR/BCES00850-%5BLittleBigPlanet%E2%84%A22%20(EU)%5D.log#L615 Found it, thanks.

**Comment by Zangetsu38** [comment]
oh yes ^^

File: rpcs3/Emu/Cell/Modules/cellSysmodule.cpp [hunk 1]
```diff
@@ -120,6 +120,7 @@ static const char* get_module_name(u16 id)
 	case 0xf030: return "cellSysutilAvc2";
 	case 0xf034: return "cellSailRec";
 	case 0xf035: return "sceNpTrophy";
+	case 0xf044: return "cellSysutilNpEula";
```
Comment (Zangetsu38) [inline rpcs3/Emu/Cell/Modules/cellSysmodule.cpp]: Probably best choice set "sceNpEula" same other NP function

Comment (AniLeo) [inline rpcs3/Emu/Cell/Modules/cellSysmodule.cpp]: @Zangetsu38 nope, the correct name for the module is that one. If you want to change it you'd have to ask sony engineers.

Comment (Zangetsu38) [inline rpcs3/Emu/Cell/Modules/cellSysmodule.cpp]: ah ok, just other Np function have sceNP..., but if you say ok ^^

**Pull request merged**